kola/tests/misc/ntp.go

```diff
@@ -1,68 +0,0 @@
-// Copyright 2015 CoreOS, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package misc
-
-import (
-	"bytes"
-	"fmt"
-	"time"
-
-	"github.com/coreos/mantle/kola/cluster"
-	"github.com/coreos/mantle/kola/register"
-	"github.com/coreos/mantle/util"
-)
-
-func init() {
-	register.Register(&register.Test{
-		Run:              NTP,
-		ClusterSize:      0,
-		Name:             "linux.ntp",
-		Platforms:        []string{"qemu"},
-		ExcludePlatforms: []string{"qemu-unpriv"},
-		Distros:          []string{"cl"},
```

Comment:
Is there a reason to not test this on fcos?

Reply:
FCOS just ships the client, not nfs-server (https://github.com/coreos/fedora-coreos-tracker/issues/121)

Reply:
Having a general NTP test would be good, however this specific NTP test isn't usable on FCOS and would require an entire re-write.